src/libraries/System.Private.CoreLib/src/System/Globalization/TextInfo.cs

```diff
@@ -461,6 +461,47 @@ internal static char ToUpperAsciiInvariant(char c)
             return c;
         }
 
+        public Rune ToLower(Rune r)
+        {
+            // Convert rune to span
+            Span<char> chars = stackalloc char[2];
+            int charsWritten = r.EncodeToUtf16(chars);
+            ReadOnlySpan<char> charsSlice = chars[..charsWritten];
+
+            // Change span to lower and convert to rune
+            if (charsSlice.Length == 2)
+            {
+                Span<char> lowerChars = stackalloc char[2];
+                ChangeCaseToLower(charsSlice, lowerChars);
```

Comment:
Using `ChangeCaseToLower()` directly here will bypass the check `if (GlobalizationMode.Invariant)` which is wrong. You may create a helper method similar to `ToLower(string)` which take a span parameter instead and call it here.